src/applications/vaos/tests/e2e/page-objects/TypeOfEyeCarePageObject.js

```diff
@@ -9,7 +9,13 @@ export class TypeOfEyeCarePageObject extends PageObject {
   }
 
   selectTypeOfEyeCare(label) {
```

Comment:
This would be a good method to add to the PageObject.

```
selectRadioButtonShadow({label}) {
  cy.get('va-radio')
      .shadow()
      .get('va-radio-option')
      .contains(label)
      .click();

    return this;
}
```